inst/mappings/mapping_ARIADNE.csv

```diff
@@ -1882,14 +1933,49 @@ Tier;Category;variable;unit;Definition;piam_variable;piam_unit;piam_factor;Comme
+2;energy (transport);Investment|Infrastructure|Transportation|Rail;billion EUR2020/yr;Total investment spending into transport infrastructure (all related to railway). Please specify in the comments section the type of infastructure that is being referred to here.;;;;;0;;
+1;energy (transport);Investment|Infrastructure|Transportation|Rail|ICE;billion EUR2020/yr;Total investment spending through Infrastructure in the transport sector (rail, internal combustion engine technologies running on any type of liquid or gaseous fuel, including mild hybrids and excluding plug-in hybrids);;;;;0;;
+1;energy (transport);Investment|Infrastructure|Transportation|Rail|BEV;billion EUR2020/yr;Total investment spending through Infrastructure in the transport sector (rail, electric vehicle technologies, only all-electrics);;;;;0;;
+1;energy (transport);Investment|Infrastructure|Transportation|Rail|FCEV;billion EUR2020/yr;Total investment spending through Infrastructure in the transport sector (rail, fuel cell technologies running on hydrogen or another type of fuel);;;;;0;;
+2;energy (transport);Investment|Infrastructure|Transportation|Bus;billion EUR2020/yr;Total investment spending into transport infrastructure (all related to buses). Please specify in the comments section the type of infastructure that is being referred to here.;;;;;0;;
+1;energy (transport);Investment|Infrastructure|Transportation|Bus|ICE;billion EUR2020/yr;Total investment spending through Infrastructure in the transport sector (bus, internal combustion engine technologies running on any type of liquid or gaseous fuel, including mild hybrids and excluding plug-in hybrids);;;;;0;;
+1;energy (transport);Investment|Infrastructure|Transportation|Bus|BEV;billion EUR2020/yr;Total investment spending through Infrastructure in the transport sector (bus, electric vehicle technologies, only all-electrics);;;;;0;;
+1;energy (transport);Investment|Infrastructure|Transportation|Bus|FCEV;billion EUR2020/yr;Total investment spending through Infrastructure in the transport sector (bus, fuel cell technologies running on hydrogen or another type of fuel);;;;;0;;
+2;energy (transport);Investment|Infrastructure|Transportation|Road;billion EUR2020/yr;Total investment spending into transport infrastructure (all related to roads). Please specify in the comments section the type of infastructure that is being referred to here.;;;;;0;;
+1;energy (transport);Investment|Infrastructure|Transportation|Charging|Public;billion EUR2020/yr;Total investment spending into public charging infrastructure.;;;;;0;;
+1;energy (transport);Investment|Infrastructure|Transportation|Charging|Private;billion EUR2020/yr;Total investment spending into private charging infrastructure;;;;;0;;
```

Comment:
I think I already mapped this variable when it had a different name.

Reply:
Nvm, will extend the script.

Reply:
I brought back the mapping for "Energy Investments|Electricity|Grid|+|BEV Chargers" , but did not find any others